components/automate-dex/web/templates/password.html

```diff
@@ -11,18 +11,13 @@
     <link href="{{ url "theme/styles.css" }}" rel="stylesheet">
     <link rel="stylesheet" href="{{ url "static/font-awesome-4.7.0/css/font-awesome.min.css" }}">
   </head>
-  <script>
-    function goBack() {
-      window.history.back();
-    }
-  </script>
   {{ if .BackLink }}
-  <div class="back" onclick="goBack()">
+  <a class="back" href="javascript:history.back()">
```

Comment:
using an anchor instead of a div means it will be announced correctly and we get mouse and focus for free